cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -1,27 +1,5 @@
 {% extends 'layout-2-1-bleedbar.html' %}
 
-{% if not page %}
-    {% do global_dict.update({
-            'page':  {
-                'seo_title':     'Sublanding Page - This is a prototype page for testing page layouts.',
-                'related_posts': global_dict.related_posts_function
-            },
-            'related_posts_settings': {
-                'value': {
-                    'limit':            3,
-                    'show_heading':     true,
-                    'related_posts':    false,
-                    'related_newsroom': false,
-                    'related_events':   false,
-                }
-            },
-            'prototype': true
-        })
-    %}
-    {% set page = page or global_dict.page %}
-    {% set prototype = global_dict.prototype or false %}
-{% endif %}
-
 {% import 'molecules/related-posts.html' as related_posts with context %}
 {% import 'organisms/post-preview.html' as post_preview %}
```

Comment:
since prototype data has been removed these imports
```
{% import 'organisms/post-preview.html' as post_preview %}
{% import 'organisms/sidebar-contact-info.html' as sidebar_contact_info %}
{% import 'organisms/main-contact-info.html' as main_contact_info %}
```
are unncessary and causing page errors